.github/workflows/release.yaml

```diff
@@ -299,6 +316,45 @@ jobs:
 
       - name: Create release
         env:
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GH_TOKEN: ${{ steps.app-token.outputs.token }}
         run:
           gh release create ${{ github.ref_name }} --verify-tag --generate-notes dist/*
+
+  dispatch:
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    needs: github_release
+    strategy:
+      fail-fast: false
+      matrix:
+        target-branch: [release/v2.8, release/v2.9, release/v2.10, main]
```

Comment:
Are we actively updating 2.8 with new system-agents? If not, removing it might help keep the number of PRs raised to a minimum.

Reply:
Removed `release/v2.8`